### DISCUSSION TITLE
依赖已经安装crypto-js执行脚本还是提示未安装

### DISCUSSION THREAD

**Discussion by KingFuRong** [whyour/qinglong#2091] [Q&A]
![IMG_20230826_223645_726.jpg](https://github.com/whyour/qinglong/assets/119987402/bdadc846-cb7c-408a-a347-15bf5c3c64c4)

![IMG_20230826_223607_743.jpg](https://github.com/whyour/qinglong/assets/119987402/da8d8e20-80af-436c-b9b2-af6f8e112795)

**Comment by whyour** [comment]
> ![IMG_20230826_223645_726.jpg](https://github.com/whyour/qinglong/assets/119987402/bdadc846-cb7c-408a-a347-15bf5c3c64c4)
> 
> ![IMG_20230826_223607_743.jpg](https://github.com/whyour/qinglong/assets/119987402/da8d8e20-80af-436c-b9b2-af6f8e112795)
> 
> 

重启容器试试，更新镜像到最新版
